<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AjouterIngredientDialog</class>
 <widget class="QDialog" name="AjouterIngredientDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="lineEditNom">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>221</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxQuantite">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>63</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxUnite">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>180</y>
     <width>82</width>
     <height>28</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>g</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>kg</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ml</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>L</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>pièce</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom d'ingredient</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Quantité</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>unité</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>282</width>
     <height>64</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>57</number>
    </property>
    <property name="leftMargin">
     <number>11</number>
    </property>
    <property name="topMargin">
     <number>15</number>
    </property>
    <property name="rightMargin">
     <number>54</number>
    </property>
    <property name="bottomMargin">
     <number>13</number>
    </property>
    <item>
     <widget class="QPushButton" name="btnValider">
      <property name="text">
       <string>Valider</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnAnnuler">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Annuler</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
